lib/text.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+
+class Text
+
+  def initialize(client = Twilio::REST::Client)
+    @message = "Thank you! Your order was placed. The total was: "
+    @account_sid = ENV['ACCOUNT_ID']
+    @auth_token = ENV['AUTH_TOKEN']
+    @from = ENV['TWILIO_PHONE']
+    @to = ENV['MY_PHONE']
+    @client = client.new @account_sid, @auth_token
+  end
+
+  def send(total)
```

Comment:
So the user only gets to see the total in the confirmation?  Would you be happy with this?